examples/mosaic-integration/src/vizzes/Earthquakes.ts

```diff
@@ -0,0 +1,50 @@
+import * as vg from '@uwdata/vgplot';
+import * as topojson from 'topojson-client';
+import { Viz } from '../Viz';
+
+// Requires the MotherDuck sample_data database to be present.
+// It is automatically shared to all new MotherDuck accounts, but if you've detached it you can get it back with the following:
+// ATTACH 'md:_share/sample_data/23b0d623-1361-421d-ae77-62d701d471e6';
+// More details: https://motherduck.com/docs/sample-data-queries/attach-sample-database/
```

Comment:
The comments in all three of new new viz files should follow the one in FlightsViz.ts, not NYPDComplaintsViz.ts.

In particular, these do (or should) depend on the `mosaic_examples` share, not the `sample_data` one.